README.md

```diff
@@ -1,41 +1,43 @@
 # FAQ
 
-## What is patcher.xyz?
+## General
+
+### What is patcher.xyz?
 
 ![Patcher Promo Image](.gitbook/assets/patcher_promo_4.png)
 
-This project solves the problem of transcribing and sharing patch notes to save them or simply to send them to someone. No more typing!
+Patcher.xyz is a platform designed to simplify the process of transcribing and sharing patch notes. It allows users to build patches by clicking the INs and OUTs of modules in the "Patch Builder", visualize the signal flow, and share or save the patches with ease. It also features a rack builder for sorting modules.
 
-Patches are built by clicking the INs and OUTs of modules added to your collection in the "Patch Builder". You can then see the signal flow in our unique graph visualizer.
+## Usage
 
-Similarly to modulargrid, we also have a rack builder that can sort modules while dragging and dropping.
+### What can I do with the patch builder?
 
-## What can I do?
+The patch builder is a versatile tool that allows you to connect CV inputs and CV outputs quickly and easily. For instance, you can create a complex signal flow by connecting multiple modules, visualize it in our unique graph visualizer, and then share the URL or save the list of connections as text. All patches are saved on the website and can be shared with a simple link.
 
-The patch builder allows connecting CV inputs and CV outputs in an easy and fast way.
+### Do I need a list of all modules and their information to use the patch builder?
 
-You can then share the URL or save the list of connections as text. Patches are saved on the website itself and can be shared with a simple link.
+Yes, but don't worry! We already have a substantial database of inputs and outputs for each module, thanks to the contributions of our community. We're always looking for more help to map all modules' CVs.
```

Comment:
Add something about the fact that we ask for the help of the community to map all modules' CVs. We already have a lot of data mapped and ready.